fec.config.js

```diff
@@ -26,15 +27,23 @@ module.exports = {
     }),
   },
   moduleFederation: {
-    moduleName: 'widgetLayout',
+    moduleName: process.env.CONFIG_PORT || process.env.PROXY ? 'frontendStarterApp' : 'widgetLayout', // overwriting starter app locally for ease of development
     exposes: {
       './RootApp': path.resolve(__dirname, './src/AppEntry.tsx'),
+      // TODO these will likely need to be restructured to be more consumable
+      './WidgetLayout': path.resolve(__dirname, './src/Components/DnDLayout/GridLayout.tsx'),
+      './WidgetDrawer': path.resolve(__dirname, './src/Components/WidgetDrawer/WidgetDrawer.tsx'),
+      './WidgetHeader': path.resolve(__dirname, './src/Components/Header/Header.tsx'),
     },
-    exclude: ['react-router-dom', 'jotai'],
+    exclude: ['@patternfly/react-core', 'jotai', 'react', 'react-dom', 'react-redux', 'react-router-dom'],
     shared: [
       {
-        'react-router-dom': { singleton: true, requiredVersion: '*' },
+        '@patternfly/react-core': { singleton: true, requiredVersion: dependencies['@patternfly/react-core'] },
         jotai: { singleton: true, requiredVersion: '*' },
```

Comment:
Jotai should be shared as a singleton ever.

Reply:
@Hyperkid123 without specifying jotai as singleton, the following warning appears in the browser console:
`Detected multiple Jotai instances. It may cause unexpected behavior with the default store`. 

For now I was using the default context. In the future I'd like to use a context scoped to just widgets.